docs/developing/lit-html.md

```diff
@@ -0,0 +1,14 @@
+# lit-html
```

Comment:
Should this be LitElement instead? Since LitElement is the wc base class that uses lit-html? It feels a little "omgekeerde wereld" if that makes sense

(also i realize this a bit nitpicky)

Reply:
My reasoning is that `lit-html` is the library, and you use `lit-element` a base class that implements it but most of the magic is in `lit-html`. People most often speak of lit-html, and not lit-element.

Reply:
You OK with discussing this separately from this PR?